connect-plugins/processor/embeddings/openai/README.adoc

```diff
@@ -28,7 +28,7 @@ NOTE: Make sure your account has https://help.openai.com/en/articles/9038407-how
 
```

Comment:
@rockwotj please could you add this to the header of this doc. It'll generate a redirect for the old `benthos-plugins` URL:

```
:page-aliases: benthos-plugins:openai.adoc
```

Reply:
I'll add this as a follow-up - thanks!